windows/task.go

```diff
@@ -284,15 +281,14 @@ func (t *task) newProcess(ctx context.Context, id string, conf *hcsshim.ProcessC
 			}
 		}()
 	}
-	t.Unlock()
```

Comment:
the unlock could just be a `defer`, not sure why I had it that way :man_shrugging: